README.md

```diff
@@ -1 +1,308 @@
-# measurement-opt
\ No newline at end of file
+# Measurement This, Not That: Optimizing the Cost and Model-Based Information Content of Measurements
+
+Authors: Jialu Wang, Zedong Peng, Ryan Hughes, Debangsu Bhattacharyya, David E. Bernal Neira, Alexander W. Dowling 
+
+This repository contains code and results for paper: Measure this, not that: Optimizing the cost and model-based information content of measurements 
+
+## Installation instructions 
+
+The following instructions assume you have anaconda installed. We suggest create an environment with the following commands to run code: 
+
+### Step 1: create a new environment 
+- create new environment, called for e.g. `measurement_optimization`, with `conda` with `Python` version 3.8
+
+`conda create --name measurement_optimization python=3.8`
+
+`conda activate measurement_optimization`
+   
+### Step 2: install `IDAES-PSE`
+- this step provides `Ipopt` solver, but this solver is not necessary for reproducing paper results
```

Comment:
Please test this with a new environment without IDAES and update the instructions

Reply:
Other packages should include `numpy` and `pickle`